Directory.Build.targets

```diff
@@ -9,4 +9,11 @@
     <PublishTrimmed Condition=" '$(TargetFramework)' == 'net6.0-ios' or '$(TargetFramework)' == 'net6.0-maccatalyst' ">false</PublishTrimmed>
   </PropertyGroup>
 
-</Project>
+  <Target Name="Net6WinUIPackagingWorkaround" BeforeTargets="GetPackagingOutputs" Condition="$(TargetFramework.Contains('-windows')) == true AND '$(DisableEmbeddedXbf)' != 'true' ">
```

Comment:
Does this only need to happen on Pack?

Reply:
Yea we only want it to pack into the pri when you're building nugets 

Otherwise it slows down build performance while doing development